src/components/CardCocktailList.jsx

```diff
@@ -0,0 +1,62 @@
+import { Button, Card, CardBody, Center, Collapse, Container, Heading, HStack, Stack, Tag, TagLabel, Image } from '@chakra-ui/react';
+import { useState } from 'react';
+
+const CardCocktailList = ({ strDrink, strAlcoholic, strIngredient1, strDrinkThumb }) => {
+  const [isShown, setIsShown] = useState(false);
+  const ingredients = [strAlcoholic, strIngredient1];
+  return (
+    <div>
+      <Card
+        onMouseEnter={() => setIsShown(true)}
+        onMouseLeave={() => setIsShown(false)}
+        bg='light.50'
+        backdropFilter='saturate(200%)'
+        margin='0.5rem'
+      >
+        <CardBody maxWidth='280px' padding='0'>
+          <Image borderRadius='lg' src={strDrinkThumb} alt={strDrink} zIndex={1} filter='auto' brightness='95%' padding='8px' />
+          <Container
+            display={isShown ? 'none' : 'flex'}
```

Comment:
Non
Tu peux:
rajouter `role='group'` sur la Card
```suggestion
            display={'flex'} {/* état par défaut, c'est à dire isShown à false */}
            _groupHover={{ display: 'none' }} {/* quand l'element avec role=group sera hover, ça va trigger ceci */}
```